dist/stat_data.py

```diff
@@ -539,6 +543,7 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     YieldStat('application_cache_time', 'application thread time waiting for cache (usecs)'),
     YieldStat('application_evict_time', 'application thread time evicting (usecs)'),
+    YieldStat('blocked_for_prepare_transition', 'waiting for prepare state transition stabilization'),
```

Comment:
I prefer `prepared_transaction_blocked_page` description `page access yielded due to prepared state change`